samples/azure/blob-storage-databus/ABSDataBus_1/Receiver/Program.cs

```diff
@@ -16,7 +16,8 @@ static void Main()
         endpointConfiguration.UseSerialization<JsonSerializer>();
         var dataBus = endpointConfiguration.UseDataBus<AzureDataBus>();
         dataBus.ConnectionString("UseDevelopmentStorage=true");
-        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.UseTransport<LearningTransport>();
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```

Comment:
it is required, unless we disable timeout manager, but then retries won't be available, so I'd rather leave as is.